projects/standard-rulesets/src/spectral/index.ts

```diff
@@ -87,6 +94,55 @@ export class SpectralRulesets extends ExternalRuleBase {
 
     return allResults.flat(1);
   }
+
+  async runRulesV2(inputs: {
```

Comment:
Again, also copy pasted from runRules and slightly modified for different inputs / outputs